pkg/config/leaderelection/leaderelection.go

```diff
@@ -64,7 +65,8 @@ func ToConfigMapLeaderElection(clientConfig *rest.Config, config configv1.Leader
 		RetryPeriod:     config.RetryPeriod.Duration,
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStoppedLeading: func() {
-				klog.Fatalf("leaderelection lost")
+				defer os.Exit(0)
```

Comment:
you like defer, I repeat myself.

Reply:
this is right way :) you want os exit in defer so other defers will run... it is good to get a habit ;)